src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.android.cs

```diff
@@ -49,9 +59,9 @@ static class DrawingViewService
 			var drawingHeight = points.Max(p => p.Y) - minPointY;
 			const int minSize = 1;
 			if (drawingWidth < minSize || drawingHeight < minSize)
-				return null;
+				throw new Exception("Image is too small");
```

Comment:
Should add some suggestion, like "please provide a bigger image" or "the image should have at least ${height} x ${width}" ?
```suggestion
				throw new Exception("Image is too small.");
```